buildbot/configure.py

```diff
@@ -101,6 +109,15 @@ def do_configure(args):
     # Add path to root CMakeLists.txt
     cmake_cmd.append(llvm_dir)
 
+    if args.use_libcxx:
+      if not (args.libcxx_include and args.libcxx_library):
+        sys.exit("Please specify include and library path of libc++ when building sycl "
+                 "runtime with it")
```

Comment:
Is it a strong requirement? Can we fallback to libc++, that resides in this source tree?

Reply:
If we need to fallback to libc++, we introduce the dependency of libcxx which means we need to build libc++ and libc++abi. Since building sycl runtime with libcxx is experimental feature, requiring users to provide a stable pre-built libcxx simplifies our work.
Thanks very much.

Reply:
Ok, agreed